recipes/onnxruntime/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "onnxruntime" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/microsoft/{{ name }}
```

Comment:
Could you please use tarballs instead of ``git_url`` ?

Reply:
```suggestion
  url: https://github.com/microsoft/onnxruntime/archive/v{{ version }}.tar.gz
```

Reply:
It's not as simple. onnxruntime have a lot of submodules for external tools. We could reference and download them separetely, but it would be harder to maintain.

Can we keep git source ?

Reply:
What is the difference since that release is the source code?

and please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

Reply:
I understand why we should use tarball instead of git.

But here, onnxruntime build system use git submodules for external tools:
https://github.com/microsoft/onnxruntime/blob/068b568472bab389aa6372c27a752638f5a26eb3/tools/ci_build/build.py#L194

```
$ git submodule status
 ab22710a3f0166f31c9c14feab98c04bfb86b71b cmake/external/DNNLibrary (v0.7.2-24-gab22710)
 e7e1482087f58913b80a20b04d5c58d9d6d90155 cmake/external/date (v2.4.1)
 1456fe2fac0cd074bbbb8630ba003a7c325d015b cmake/external/eigen (3.3.0-1740-g1456fe2fa)
 42c53187a56c12dc5518cc25e778e5e3e7dbaf72 cmake/external/gemmlowp (remotes/origin/revert-87-master-139-g42c5318)
 9bda90b7e5e08c4c37a832d0cea218aed6af6470 cmake/external/googletest (release-1.8.0-712-g9bda90b7)
 02a2a458ac15912d7d87cc1171e811b0c5219ece cmake/external/grpc (v1.12.1-4734-g02a2a458ac)
 cee3125af7208258d024a75e24f73977eddaec5b cmake/external/gsl (v1.0.0-10-gcee3125)
 8f50e4463c2c7ba9b3f580c61ca21abc91197b7c cmake/external/nsync (1.20.1-1-g8f50e44)
+7d90796473295ca3cdf976ed772215c5980ad3e0 cmake/external/onnx (v1.3.0-340-g7d907964)
+6c37109733a9bbf8211f0ca78a85804cb376eca0 cmake/external/onnx-tensorrt (6c37109)
 48cb18e5c419ddd23d9badcfe4e9df7bde1979b2 cmake/external/protobuf (v3.6.0.1-37-g48cb18e5)
 30cad267151fa8f1b17da8c1ef0571da6da9a8f1 cmake/external/re2 (2019-01-01-17-g30cad26)
 352281313fe1c4313bc222cb9de222afd50c822f cmake/external/spdlog (v1.2.1-496-g35228131)
 fd4801612817f96e890058656834deb925fc064a cmake/external/tvm (v0.4-663-gfd480161)
```

All this external tools are not in the tarball release.

Reply:
Almost all of those packages we do have them on ``conda-forge``
``tvm``, I was trying to create a package for it
``onnx``, ``eigen``, ``googletest`` and a lot of them they are already being packaged by ``conda-forge``

Reply:
So, basically it is missing ``tvm`` and ``mkl-dnn``

Reply:
``tvm`` I stopped because they are going to launch the version ``0.6`` which will be easier to package it on ``conda-forge``
I am waiting their new release

Reply:
I can try to use conda-forge packages for this dependencies but I will have to onnxruntime build script and it will be harder to keep versions sync.

Is it a good idea to do that ?

Reply:
It is better, we tried to isolate and package just the project itself. Each dependency will be a separated feedstock